pkg/cmd/cli/cli.go

```diff
@@ -117,7 +117,8 @@ func NewCommandCLI(name, fullName string) *cobra.Command {
 			Message: "Advanced Commands:",
 			Commands: []*cobra.Command{
 				cmd.NewCmdCreate(fullName, f, out),
-				cmd.NewCmdUpdate(fullName, f, out),
```

Comment:
> is oc update still going to work (with a deprecation warning) with this change?

Yes, it works via an alias registered for the cmd in cobra.  That happens in kube code.  `test-cmd.sh` ensures it keeps working.  See https://github.com/openshift/origin/pull/3625/files#diff-e2c8544666bb21f92286b190d5c8cb9aR320